### ISSUE TITLE
Cursor doesn't respect `highlight Cursor` setting

### ISSUE THREAD

**Issue by haifengkao** [rhysd/neovim-component#31]
see http://vim.wikia.com/wiki/Configuring_the_cursor

`NeovimCursor` should store the char under the cursor to redraw it correctly. It can avoid the ugly anti-aliasing issue as well.

**Comment by rhysd** [comment]
> NeovimCursor should store the char under the cursor to redraw it correctly. It can avoid the ugly anti-aliasing issue as well.

Yes, as you said, it's the best that `NeovimCursor` contains the character under the cursor.  It was discussed once in NyaoVim issue.

https://github.com/rhysd/NyaoVim/issues/16

However, I could not find the way to obtain the character under the cursor efficiently (Note: RPC is slow).  So currently neovim-component stupidly inverts the color under the cursor.
